generated-docs/FRP/Event.md

```diff
@@ -62,21 +62,32 @@ filter :: forall a. (a -> Boolean) -> Event a -> Event a
 
 Create an `Event` which only fires when a predicate holds.
 
+#### `keepLatest`
+
+``` purescript
+keepLatest :: forall a. Event (Event a) -> Event a
+```
+
+Flatten a nested `Event`, reporting values only from the most recent
+inner `Event`.
+
 #### `subscribe`
 
 ``` purescript
-subscribe :: forall eff a r. Event a -> (a -> Eff (frp :: FRP | eff) r) -> Eff (frp :: FRP | eff) Unit
+subscribe :: forall eff a r. Event a -> (a -> Eff (frp :: FRP | eff) r) -> Eff (frp :: FRP | eff) (Eff (frp :: FRP | eff) Unit)
 ```
 
 Subscribe to an `Event` by providing a callback.
 
-#### `create`
+`subscribe` returns a canceller function.
```

Comment:
My initial thought about this is whether it would be useful to have access to this cancellation function _within_ the `subscribe` callback, but I'm now thinking about space leaks. I'll have a better look later on!

Reply:
I still think access to this within the callback may be useful. For example, to listen for only one instance of an event:

```
subscribe e \value cancel -> do
  log "Caught one! " <> value
  cancel
```

Reply:
I think we could do this with a `take` combinator though, and I'd rather not complicate the `subscribe` function any more unless there is something we can't implement otherwise.

Reply:
Agreed. I can't think of a use case I'm likely to want that I can't do in another way, so colour me convinced!